Makefile

```diff
@@ -59,12 +59,11 @@ tests-marimo:
 	@echo "All marimo tests passed successfully."
 
 tests-marimo-mlir:
-	@if command -v mlir-opt &> /dev/null; then
```

Comment:
is `&>` not supported on mac `bash`? O.o

Reply:
that bit was fine, it's the linux that didn't like this

Reply:
for mac it was the nested `@` and missing `\`